### DISCUSSION TITLE
Docker方式启动，iOS无法播放

### DISCUSSION THREAD

**Discussion by joyoner** [1715173329/UnblockNeteaseMusic#756] [Q&A] [answered]
启动方式：
docker run -d -m 300M --memory-swap -1 -e ENABLE_LOCAL_VIP=true -e NO_CACHE=true -e ENABLE_FLAC=true -e LOG_LEVEL=debug --network=host --restart=always --log-opt max-size=10m --name=music pan93412/unblock-netease-music-enhanced -e https://music.163.com -o kugou -s -p 18080 

测试环境：iOS 证书已安装并信任

错误1：iOS客户端侧，提示歌曲不存在
错误2：后台docker debug 日志
`root@HKG-CERA:~# docker logs music -f
INFO: (app) HTTP Server running @ http://0.0.0.0:18080
DEBUG: (request) Start requesting https://music.163.com/api/song/detail?ids=[26609720]
DEBUG: (provider/match) Getting the audio...
    source: "kugou"
    info: {
      "name": "亡命追逐",
      "id": 26609720,
      "alias": [],
      "duration": 234000,
      "album": {
        "name": "天台 电影原声带",
        "id": 2489195
      },
      "artists": [
        {
          "name": "黄雨勋",
          "id": 1026086
        }
      ],
      "keyword": "亡命追逐 - 黄雨勋"
    }
DEBUG: (request) Start requesting http://mobilecdn.kugou.com/api/v3/search/song?keyword=%E4%BA%A1%E5%91%BD%E8%BF%BD%E9%80%90%20-%20%E9%BB%84%E9%9B%A8%E5%8B%8B&page=1&pagesize=10
DEBUG: (request) Start requesting http://trackercdn.kugou.com/i/v2/?key=75953fe9a0747dd294796dbd55c8033a&hash=0067ef6d696ced8d2d66103fc54b967e&appid=1005&pid=2&cmd=25&behavior=play&album_id=552046
DEBUG: (request) Start requesting http://trackercdn.kugou.com/i/v2/?key=dbf77c85e7291d634b09014632243b71&hash=96956d3c6a3bc7ef0a422cbf1a2e5b04&appid=1005&pid=2&cmd=25&behavior=play&album_id=552046
DEBUG: (request) Start requesting http://trackercdn.kugou.com/i/v2/?key=a6164359d6ed72c34ed3c004de9a5e56&hash=6d1bd391e4e11b809ce3ea730843891a&appid=1005&pid=2&cmd=25&behavior=play&album_id=552046
ERROR: (hook) All promises were rejected
    AggregateError: All promises were rejected`

以上错误日志重复出现

**Comment by 1715173329** [comment, marked as answer]
所以并不建议用单个音源。
在 kugou 这首歌曲是收费的，自然无法获取信息。
